sopel/coretasks.py

```diff
@@ -1,9 +1,17 @@
 # coding=utf-8
-"""Tasks that allow the bot to run, but aren't user-facing functionality
+"""Core Sopel plugin that listen to basic IRC messages.
```

Comment:
```suggestion
"""Core Sopel plugin that listens to basic IRC messages.
```

But actually, I'd go further:

```suggestion
"""Core Sopel plugin that handles IRC protocol functions.
```